src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -373,11 +367,11 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
     val expected = AppInfo(
       app.copy(
         versionInfo = AppDefinition.VersionInfo.OnlyVersion(clock.now()),
-        container = Some(app.container.get.copy(docker = Some(app.container.get.docker.get.copy(
+        container = Some(app.container.get.asInstanceOf[Container.Docker].copy(
```

Comment:
Alright, but then I need to write extra code to make the test fail. I think that asInstanceOf is great for testing!